client/actions/actions.js

```diff
@@ -0,0 +1,137 @@
+import * as ActionTypes from './actionTypes';
```

Comment:
This file contains all of the redux action creators, you'll see the react app dispatching these actions